tools/generator/src/Generator/AddBazelDependenciesTarget.swift

```diff
@@ -104,13 +106,41 @@ $(BAZEL_INTEGRATION_DIR)/calculate_output_groups.py
                 filePathResolver: filePathResolver
             )
 
+        var buildPhases = [
```

Comment:
Doing this here will make it that these run scripts won't apply to BwX. Is that something we want to support as well?